src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/UserRoleController.kt

```diff
@@ -149,13 +151,18 @@ class UserRoleController(
     @PathVariable
     role: String,
   ) {
-    userRoleService.removeRoleByUserId(userId, role)
+    if (authenticationFacade.getAuthentication().authorities.map { it.authority }.any { it == "ROLE_MAINTAIN_IMS_USERS" }
+    ) {
+      userRoleService.serviceRemoveRoleByUserId(userId, role)
```

Comment:
bit of a naff name for a service method - doesn't distinguish between that and the one below.  Would be good to add some javadoc to service methods too explaining which to call and when

Reply:
yep - removed service method!